main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -69,10 +71,22 @@ object Main {
       case Some(s) => Some(Symbol.mkDefnSym(s))
     }
 
-    // Get GitHub token
+    // get GitHub token
     val githubToken =
       cmdOpts.githubToken
+        .orElse(FileOps.readString(cwd.resolve("./.GITHUB_TOKEN")))
         .orElse(sys.env.get("GITHUB_TOKEN"))
+        .map { t =>
+          val trimmed = t.trim
```

Comment:
This code here breaks the flow and seem unnatural. May I instead suggest to have FileOps.readLine which reads only the first line?

Reply:
I agree, but secretly only reading the first line sounds a bit unpredictable. What about moving this part to a helper function called something like `sanitizeGithubToken`?

Reply:
I think simple-stupid is best. What's wrong with reading just the first line?

Reply:
I really dont think we want to start doing input validation on the token.